Linear regression

Hrabousk opened the pull request:
linear regression более-менее причёсана, а вот logistic лучше не смотреть пока

Hrabousk committed:
Add files via upload

Hrabousk committed:
Add files via upload

IlyaBY-afk reviewed (approved):
Хорошее решение, красивый график, мне нравится кодстайл. Хотелось только увидеть пару экспериментов с гиперпараметрами

Hrabousk committed:
Merge branch 'TevenixLevelUps:main' into main

Hrabousk committed:
Add files via upload

I don't know how to fix the confidence thing really. Also accuracy is rly high, hope no overfitting? Checked on different ranges of data

IlyaBY-afk commented:
По логистической регрессии: в целом всё правльно. Рекомендую лишь побольше использовать numpy, чтобы было поменшье строчек с list comprehension. Учитывая, что датасет ты генеришь рандомными точками, confidence 54% – это норма.

Hrabousk commented:
> По логистической регрессии: в целом всё правльно. Рекомендую лишь побольше использовать numpy, чтобы было поменшье строчек с list comprehension. Учитывая, что датасет ты генеришь рандомными точками, confidence 54% – это норма.

а что ты имеешь в виду, прости за непонимание, под list comprehension?

IlyaBY-afk commented:
Списковое включение. Например, b= [x ** 2 for x in a]

Hrabousk commented:
я тебя понял. Да, ты прав, спасибо

Hrabousk committed:
Add files via upload

Код сломан, и как чинить его я не знаю.

Hrabousk committed:
Update logistic regression.py
